src/analyses/libraryFunctions.ml

```diff
@@ -378,6 +408,9 @@ let invalidate_actions = [
     "BZ2_bzBuffToBuffDecompress", writes [3;4]; (*keep [3;4]*)
     "uncompress", writes [3;4]; (*keep [3;4]*)
     "stat", writes [1]; (*keep [1]*)
+    "__xstat", writes [1]; (*keep [1]*)
+    "__lxstat", writes [1]; (*keep [1]*)
```

Comment:
signature of `__xstat`, `__lxstat`, accoding to [this](https://refspecs.linuxbase.org/LSB_3.0.0/LSB-PDA/LSB-PDA/baselib-xstat-1.html): 
```
int __xstat(int ver, const char * path, struct stat * stat_buf);
int __lxstat(int ver, const char * path, struct stat * stat_buf);
```
Both functions write their third argument.

Reply:
Good catch! I wrongly copied from `stat` (which was incorrect in the first place as well)